test/test_tableMapper.js

```diff
@@ -0,0 +1,55 @@
+var assert = require('assert');
+var mapper = require('../mappers/tableMapper');
+var mongoose = require('mongoose');
+const APP_DB = 'mongodb://127.0.0.1/eventplanner_db';
```

Comment:
Overall this test case is quite hard to follow since all the values are hardcoded, would be more readable if a test table object was defined like other unit tests.